### PR TITLE
Preserve data

### PR DESCRIPTION
**Pull request by yemidada**
In this milestone, I
- will be saving contact data to the local storage.
- will be loading any data from the local storage into the input fields.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -273,13 +273,15 @@ <h1 class="title">Contact me</h1>
...
+        <form id="contact-form" action="https://formspree.io/f/xknayrnl" method="POST">
+            <input class="contact-form-name" required placeholder="Name" maxlength="30" name="name" type="text" value="Lorem Ipsum" />
+            <input class="contact-form-email" required placeholder="Email" name="email" type="email" value="loremipsum@gmail.com" />
+            <textarea class="contact-form-message"  rows="6" placeholder="Write your message here..." maxlength="500" required></textarea>
             <div class="contact-button">
-                <button class="button-outline">Get in touch</button>
+                <button id="contact-submit-btn" class="button-outline">Get in touch</button>
             </div>
+
+            <div id="error-message"></div>
```
Comment (wayungi) [inline index.html:284]: - [ ] Please note that after filling out the form, and not submitting it, the entered values should be saved in the browser 😄 
from

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @yemidada , 

image


**Congratulations on the good job done so far!** 🤝 

There are some issues that you still need to work on to go to the next project but you are almost there! 😄 

## Highlights ⚡ ⚡ 
- [ ] Correct GitHub flow. 👏🏾 
- [ ] On page reloads, the input fields are pre-filled with this data. 😄 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL]  Your Licence file is not well edited
lic

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (wayungi) [reply index.html:284]: @yemidada , there is a type in the image. Date should be data.

Comment (YoutLone) [reply index.html:284]: Hello @wayungi We don't understand completely about the date is not saved as required in your screenshot what is mean and We are confused about that.

**Review by LionRouge1 — APPROVED** [review]
Hi @team, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**